README.md

```diff
@@ -67,91 +67,20 @@ If a different username is required, add the `--user` switch:
 
 	--user user-name --instance=instance-name.dev
 
-### To an arbitrary URL
+### An arbitrary URL
 
 	medic-conf --url=https://username:password@example.com:12345
 
 ## Perform specific action(s)
 
 	medic-conf <--local|--instance=instance-name|--url=url> <...action>
 
-The list of available actions can be seen in [`supported-actions.js`](https://github.com/medic/medic-conf/blob/master/src/cli/supported-actions.js).
+The list of available actions can be seen via `medic-conf --help`.
 
 ## Perform actions for specific forms
 
 	medic-conf <--local|--instance=instance-name|--url=url> <...action> -- <...form>
 
-# Project Layout
```

Comment:
There are two sections in this doc about the command-line arguments with this section in between them. I'm just moving this lower in the file so the two sections are together.